README.md

```diff
@@ -2,7 +2,35 @@
 
 # DAZN Lambda Powertools
 
-An integrated suite of powertools for Lambda functions to make it effortless for you to comply with our guidelines around logging and monitoring:
+`dazn-lambda-powertools` is a collection of middlewares that make working with lambda easier.
+
+## Motivation
+
+Writing Lambda's often involves the bootstrapping of specific tooling, like reading and forwarding on correlation-id's, emitting logs on a lambda timeout, and more.
+
+Re-writing and maintaining this bootstrapping logic into every individual lambda can be a pain, so to prevent this re-work we created `dazn-lambda-powertools`.
+
+## Usage
+
+The quickest way to get setup is to use the opinionated [pattern basic](/packages/lambda-powertools-pattern-basic) package, like so:
+
+```javascript
+const wrap = require('@dazn/lambda-powertools-pattern-basic')
+
+module.exports.handler = wrap(async (event, context) => {
+  return 42
+})
+```
+
+For more control, you can pick and choose from the [individual packages](/packages/lambda-powertools-pattern-basic).
```

Comment:
```suggestion
For more control, you can pick and choose from the [individual packages](/packages).
```

intending to link to the packages folder?

Reply:
Oops, yep!